common/util.go

```diff
@@ -232,7 +232,7 @@ func ProfileFormatMimeType(f ffmpeg.Format) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	if m := ext2mime[ext]; m != "" {
+	if m, ok := ext2mime[ext]; ok && m != "" {
```

Comment:
The extra `m != ""` check isn't really needed but harmless I suppose.

This is OK to merge after a rebase / fixup.